.github/workflows/ci.yml

```diff
@@ -9,9 +9,9 @@ name: Continuous Integration
 
 on:
   pull_request:
-    branches: ['**']
+    branches: ['**', '!update/**']
```

Comment:
If someone is wondering about this `!`- it's not a negation here. @armanbilge has clarified it in [sbt-typelevel](https://github.com/typelevel/sbt-typelevel/pull/291#discussion_r889738142).